src/runtime/api-channel.ts

```diff
@@ -33,6 +33,7 @@ interface MappingInfo<T> {
   type: MappingType;
   initializer?: boolean;
   redundant?: boolean;
+  reload?: boolean;
```

Comment:
This pertains to the initialiser, so shouldn't this be 'overriding' instead of 'reload'?

Reply:
Changed!